src/URLs.ts

```diff
@@ -0,0 +1,26 @@
+const HTTP_PREFIX = /^http(s?):*\/\//i
+
+export function sanitizeURL(url: string | null | undefined): string {
+    if (!url) {
+        return 'http://'
+    }
```

Comment:
I am wondering if this should return an empty string. Would the editor handle well a link like `http://`?

Reply:
yes good question, I'm not sure at all why it does that.  I'll just return whatever it was there